db/migrate/20170316122243_create_events.rb

```diff
@@ -0,0 +1,12 @@
+class CreateEvents < ActiveRecord::Migration
+  def change
+    create_table :events do |t|
```

Comment:
So how can we know who created the event, or which organization has organised the event ? 😊